### PR TITLE
Adding registry hive retrieval for ASC-VmHealth mode.

### PR DESCRIPTION
**Pull request by waynekuo**
Adding registry hive retrieval for ASC-VmHealth mode.

### CODE REVIEW

File: pyServer/manifests/windows/asc-vmhealth [hunk 1]
```diff
@@ -1,3 +1,7 @@
 echo,### ASC Extension - Log for VM Health Signals ###
 
 copy,/WindowsAzure/Logs/TransparentInstaller.log
+
+echo,### Registry Hives ###
+copy,/Windows/System32/config/SOFTWARE
```
Comment (maheshnemichand) [inline pyServer/manifests/windows/asc-vmhealth]: Given that this increases the size of output and it is needed for the new scenario, how about we create a new mode called MinDiagnostic that starts with least amount of information for auto diagnosis.

File: pyServer/manifests/windows/min-diagnostic [hunk 2]
```diff
@@ -0,0 +1,4 @@
+echo,### EXPERIMENTAL FEATURE ###
+echo,### Registry Hives ###
+copy,/Windows/System32/config/SOFTWARE
+copy,/Windows/System32/config/SYSTEM
```
Comment (maheshnemichand) [inline pyServer/manifests/windows/min-diagnostic]: This needs to be a superset of ASC-VMHealth.

Comment (waynekuo) [reply pyServer/manifests/windows/min-diagnostic]: Fixed!

**Comment by CraigWiand** [comment]
Is there a corollary Linux version of `min-diagnostic`?  For clarity, if `min-diagnostic` is requested for Linux or BSD it will run `normal` per [this logic](https://github.com/Azure/azure-diskinspect-service/blob/master/pyServer/GuestFishWrapper.py#L324).

File: pyServer/manifests/windows/min-diagnostic [hunk 3]
```diff
@@ -0,0 +1,7 @@
+echo,### EXPERIMENTAL FEATURE ###
```
Comment (amitchat) [inline pyServer/manifests/windows/min-diagnostic]: You can remove this line.

**Comment by waynekuo** [comment]
> Is there a corollary Linux version of min-diagnostic? For clarity, if min-diagnostic is requested for Linux or BSD it will run normal per this logic.

I think this makes sense for now. Once we figure out the min set of diagnostics for Linux for the new Insight, we can add the Linux version later.

**Comment by amitchat** [comment]
LGTM

**Pull request merged**